src/components/Tag/Tag.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import theme from '../../theme';
+import { get, lodashGet } from '../../utils/utils';
+import { COMMON, COLOR } from '../../utils/constants';
+
+const TagBase = styled.div`
+    // text
+    ${get('typography.body')};
+    color: ${(props) => props.textColor};
+
+    // border
+    border-radius: 4px;
+
+    // display
+    display: inline-flex;
```

Comment:
nice

Reply:
lol just learned about this for centering text vertically